sceptre/cli/template.py

```diff
@@ -137,3 +137,32 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="stack-name", short_help="Reveal the stack name or names.")
```

Comment:
This command doesn't really seem like a _template_ command. It more appropriately seems a "list" command. It seems like the syntax would better be:

`sceptre list stack-names`